news/12918.feature.rst

```diff
@@ -0,0 +1 @@
+Deprecate '_' in version part of a wheel filename, use '-' instead.
```

Comment:
I'm not really sure what this should say (probably a simplified version of the deprecation message), but this should be renamed to `12918.removal.rst` which yes can be used for deprecations as well.

Reply:
Updated.